src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
```

Comment:
```suggestion
   private HibernateUtil() {
   }
```

Reply:
You should add private default constructor in order to prevent creating HibernateUtil object (common mistakes)